File: Models/DTOs/Story.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WorkTracker.Models.DTOs
{
    public class Story
    {
        public int StoryId { get; set; }
        public int StateId { get; set; }
        public int? ProjectId { get; set; }
        public int? SprintId { get; set; }
        public int ListOrder { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public bool? Archived { get; set; }
        public DateTime CreatedAt { get; set; }
        public int CreatedBy { get; set; }
        public DateTime? ModifiedAt { get; set; }
    }
}

File: WorkTracker/ClientApp/src/types/story.ts
import { Task } from './task';

export interface Story {
  storyId?: number;
  stateId?: number;
  projectId?: number;
  sprintId?: number;
  title?: string;
  description?: string;
  listOrder?: number;
  createdBy?: string,
  modifiedAt?: string,
  tasks?: Task[];
};
